src/octopeer-github/main/ElementEventBinding/StartWatchingPRElementEventBinding.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Created by Mathias on 2016-05-11.
+ * Class to create StartWatchingPR events
+ */
+class StartWatchingPRElementEventBinding implements ElementEventBinding {
+    public eventType: string = "startwatchingPR";
```

Comment:
This should not be an event that is hooked to the DOM. We need to register this event, but this is probably easier to do on tab update (chrome event)